runtime/onert/core/include/ir/Shape.h

```diff
@@ -71,7 +71,7 @@ struct Shape
 {
 public:
   static int32_t const UNSPECIFIED_DIM;
-  static int32_t const MAX_RANK;
+  static int32_t const ONERT_MAX_RANK;
```

Comment:
It was changed because it was duplicated with the one defined in libnpuhost.h.

Reply:
Please make a PR for this and related changes. And let us merge the PR first.

Reply:
I made the PR #9748